src/tutorial/source.rst

```diff
@@ -0,0 +1,83 @@
+.. py:currentmodule:: palace
+
+Source manipulation
+===================
+
+We have created a source in the last section.
+As said previously, its properties can be manipulated
+to create wanted effects.
+
+Moving the source
+-----------------
+
+Changing :py:meth:`Source.position` is one of the most noticeable,
+but first, we have to enable spatialization via :py:attr:`Source.spatialize`.
+
+.. code-block:: python
+
+   from time import sleep
+   from palace import Device, Context, Source, decode
+
+   filename = 'some_audio.ogg'
+   with Device() as dev, Context(dev) as ctx:
+       with Source() as src:
+           src.spatialize = True
+
+           dec = decode(filename)
+           dec.play(12000, 4, src)
+           while src.playing:
+               sleep(0.025)
+               ctx.update()
+
+Now, we can set the position of the source in this virtual 3D space.
+The position is a 3-tuple indicating the coordinate of the source.
+For those who didn't know this yet: x is for left-right, y is for up-down,
+and z is for back-forth.
```

Comment:
As per offline discussion, you know what to do.

Reply:
Do you know the name for this coordinate system? I'm not sure x is from left to right and z is from back to forth, or vice versa. Looking up 3d coordinate system only gives me systems where z is the vertical axis.

I think this part should also be explained in the doc.

Reply:
I think it's the normal system, rotated so the Oxy is on the screen, which is analogous to what how 2D graphics is drawn (so, basically, it's the 2D coordinates with Oz determined by cross product).

This is pretty much how coordinates on OpenGL works too; we only document it here because it's not obvious for many people at first<sup>[citation needed]</sup>.

Reply:
Alright, I didn't think "normal" was the name 😳

Reply:
Hey Gauss *must have*  liked it.